lefthook.yml

```diff
@@ -8,14 +8,47 @@ skip_output:
   - success
   - summary
 
-# phpcs
+# code linting
 pre-commit:
   parallel: true
-  commands:
-    phpcs:
-      run: composer run phpcs
-    phpstan:
-      run: composer run phpstan
+  jobs:
+    - name: phpcs
+      run: ./vendor/bin/phpcs -- {staged_files}
+      glob:
+        - ".phpstorm.meta.php"
+        - "local-config-sample.php"
+        - "wp-config.php"
+        - "wp-config-environment.php"
+        - "wp-content/themes/core/*.php"
+        - "wp-content/plugins/core/*.php"
+    - name: phpstan
+      run: ./vendor/bin/phpstan analyse --memory-limit=-1
+      glob:
+        - ".phpstorm.meta.php"
+        - "local-config-sample.php"
+        - "wp-config.php"
+        - "wp-config-environment.php"
+        - "wp-content/themes/core/*.php"
+        - "wp-content/plugins/core/*.php"
+    - name: lintcss
```

Comment:
Thought: I wonder if it's worth adding [the formatting scripts](https://github.com/moderntribe/moose/blob/main/package.json#L55) here, too as they are what does the prettier changes (indentations, etc).